subsys/net/ip/net_shell.c

```diff
@@ -1329,21 +1330,64 @@ static void http_server_cb(struct http_server_ctx *entry,
 	       http_str_output(output, sizeof(output) - 1,
 			       entry->req.url, entry->req.url_len));
 }
-#endif /* CONFIG_NET_DEBUG_HTTP_CONN && CONFIG_HTTP_SERVER */
+#else
+#define http_server_cb(...) NULL
+#endif
+
+static void ws_server_cb(struct ws_ctx *entry, void *user_data)
```

Comment:
Seems like we can remove some of the #if defined checks below if we create a placeholder for this function similar to http_server_cb() above.

Reply:
Indeed, I will change this.